test/libsolidity/SyntaxTest.cpp

```diff
@@ -34,17 +34,38 @@ namespace fs = boost::filesystem;
 using namespace boost::unit_test;
 
 template<typename IteratorType>
-void skipWhitespace(IteratorType& it, IteratorType end)
+void skipWhitespace(IteratorType& _it, IteratorType _end)
 {
-	while (it != end && isspace(*it))
-		++it;
+	while (_it != _end && isspace(*_it))
+		++_it;
 }
 
 template<typename IteratorType>
-void skipSlashes(IteratorType& it, IteratorType end)
+void skipSlashes(IteratorType& _it, IteratorType _end)
 {
-	while (it != end && *it == '/')
-		++it;
+	while (_it != _end && *_it == '/')
+		++_it;
+}
+
+void expect(string::iterator& _it, string::iterator _end, string::value_type _c)
+{
+	if (_it == _end || *_it != _c)
+		throw runtime_error(string("Invalid test expectation. Expected: \"") + _c + "\".");
+	++_it;
+}
+
+int parseLocation(string::iterator& _it, string::iterator _end)
```

Comment:
This isn't actually any specific about location, it just parses an unsigned number.